clients/go/test/integration_test.go

```diff
@@ -256,7 +260,11 @@ func (s *integrationTestSuite) TestStreamJobs() {
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
 
-	deployment, err := s.client.NewDeployResourceCommand().AddResourceFile("testdata/service_task.bpmn").Send(ctx)
+	taskType := uuid.NewString()
```

Comment:
:thought_balloon: Here, and in the worker test below, if we don't use a unique task type, we run the risk of receiving jobs from previous tests. This can cause things like closing the worker to block infinitely because we only consume X jobs from the dispatcher, and the handler is blocked there.

imo this is a flaw in the worker itself, but this is kind of out of scope here. Closing the worker should short circuit everything, I think.

Reply:
I think makes sense to have unique task types for each tests 👍🏼